GLOCKFILE

```diff
@@ -24,5 +24,6 @@ github.com/jteeuwen/go-bindata 7362d4b6b2ce6a7d3c28b523a6e40629f4d81116
 github.com/kisielk/errcheck eb516cb958915b69ad14384890b4d37bd910c9f3
 github.com/kisielk/gotool d678387370a2eb9b5b0a33218bc8c9d8de15b6be
 github.com/robfig/glock 78c45da050a4d993d12ad07e8ddb10a4891ac701
+golang.org/x/net 169f42267807fe93f4a4b0150583e410aefafe4e
```

Comment:
Now that we're adding this dependency we should update the imports in multiraft.go and multiraft_test.go (we were importing 	"github.com/coreos/etcd/Godeps/_workspace/src/golang.org/x/net/context". Since what we care about is an interface it should be compatible for us to pass Context objects from the real package into etcd/raft).